squirrel.worker/src/main/java/org/dice_research/squirrel/robots/RobotsManagerImpl.java

```diff
@@ -17,7 +17,7 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(RobotsManagerImpl.class);
 
     private static final String ROBOTS_FILE_NAME = "/robots.txt";
-    private static final long DEFAULT_MIN_WAITING_TIME = 100;
+    private static final long DEFAULT_MIN_WAITING_TIME = 10000;

```

Comment:
Are you sure that a default of 10 seconds is ok? Shouldn't we at least go down to 2 seconds?

Reply:
I increased in case for slow servers. Mcloud and europeandataportal sometimes were very slow

Reply:
The value is only used if the `robots.txt` file does not define this value. I think that we already talked about this and it turned out that europeandataportal has a file defining it as 10 seconds. So whatever you define here, it won't influence the crawling speed on that portal :wink: 

In general, I still think we should use something around 2 seconds. If that is too fast for the portal, it simply has to define a `robots.txt`.

Reply:
fixed both.
yaml files can be found in https://github.com/projekt-opal/squirrel-portals-config